javascript/javascript2/week2/event-application1.js

```diff
@@ -0,0 +1,15 @@
+function myFunction(numdays) {
+    const alldays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+    d = new Date()
+    today = d.getDay();
+    let num = (today+numdays) % 7
+    return "today is " + alldays[today] + " and the even will be held on " + alldays[num] + "."
+};
+
+let a = myFunction(4)
+console.log(a)
```

Comment:
1. Give meaningful variable name. Don't use alphabets like  a.  
2. You can optimize two lines(9 and 10) to one line code.
    Ex: console.log(myFunction); 
    So what is does is, myFunction is returning string and that string will be given as input to console.log().